src/app/header/header.component.html

```diff
@@ -7,5 +8,10 @@
     <div class="col-md-10 text-center">
         <span class="header-white-1 text-center">{{language?.headings.titleLeft}}</span><span class="header-2 text-center">{{language?.headings.titleRight}}</span>
     </div>
-    <button type="button" class="btn btn-default logout-button" aria-label="Left Align" (click)="logout()">{{language?.messages.logoutButton}}</button>
+  <div class="icons">
+    <i matTooltip="Update Profile" class="fa fa-pencil-square-o"></i>&nbsp; &nbsp;
+    <i matTooltip="Infokit" class="fa fa-archive" (click)="info.pop()"></i>&nbsp; &nbsp;
+    <i matTooltip="Logout" class="fa fa-power-off" (click)="logout()"></i>&nbsp;
```

Comment:
clear class names please.. referring to fa.

Reply:
@shreyans29 Sorry didn't get clearing class names.

Reply:
@shreyans29 Actually I didn't name them like that. They came with the font-awesome thing. We are required to use the same class names to use those specific icons.